metrics/src/heatmap.rs

```diff
@@ -3,16 +3,14 @@
 // http://www.apache.org/licenses/LICENSE-2.0
 
 use crate::Metric;
-use rustcommon_atomics::AtomicU64;
-use rustcommon_heatmap::AtomicHeatmap;
+// use rustcommon_atomics::AtomicU64;
+// use rustcommon_heatmap::AtomicHeatmap;
 
-pub type Heatmap = AtomicHeatmap<u64, AtomicU64>;
+// pub type Heatmap = AtomicHeatmap<u64, AtomicU64>;
```

Comment:
This commented out code should just be deleted.

Reply:
If this isn't stale, will fix.